satpy/readers/_geos_area.py

```diff
@@ -72,6 +72,7 @@ def get_area_extent(pdict):
             coff: Column offset factor
             loff: Line offset factor
             scandir: 'N2S' for standard (N->S), 'S2N' for inverse (S->N)
+            h: Altitude of satellite above the Earth's surface (m)
```

Comment:
You put `(m)` for meters, but I believe you can specify `units="km'` in PROJ.4 and `h` might be in kilometers, not meters. I'm OK merging this as it is the common case, but I did want to mention it.